### PR TITLE
[EventHub] `az eventhubs namespace application-group`: New command group to support application group management

### PR DESCRIPTION
**Pull request by damodaravadhani**
**Related command**
az eventhubs namespace application-group

**Description**
https://docs.microsoft.com/en-us/azure/event-hubs/resource-governance-overview, add cmdlets that support application groups for eventhubs

**Testing Guide**
`az eventhubs namespace application-group create --namespace-name mynamespace -g MyResourceGroup --name myAppGroup --throttling-policy-config name=policy1 metric-id=IncomingMessages rate-limit-threshold=10000
 --throttling-policy-config name=policy2 metric-id=IncomingBytes rate-limit-threshold=20000`

`az eventhubs namespace application-group create --namespace-name mynamespace -g MyResourceGroup --name myAppGroup --is-enabled false`

`az eventhubs namespace application-group show --namespace-name mynamespace -g MyResourceGroup --name myAppGroup`

`az eventhubs namespace application-group list --namespace-name mynamespace -g MyResourceGroup`

`az eventhubs namespace application-group application-group-policy add --namespace-name mynamespace -g MyResourceGroup --name myAppGroup
--throttling-policy-config name=policy1 metric-id=OutgoingMessages rate-limit-threshold=10500 |
--throttling-policy-config name=policy2 metric-id=IncomingBytes rate-limit-threshold=20000`

`az eventhubs namespace application-group application-group-policy remove --namespace-name mynamespace -g MyResourceGroup --name myAppGroup --throttling-policy-config name=policy1 metric-id=OutgoingMessages rate-limit-threshold=10500`

**History Notes**

[EventHubs] `az eventhubs namespace application-group`: New command group to support management operations on EventHubs application groups
[EventHubs] `az eventhubs namespace network-rule update`: New command to update Network Rule Sets

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
EventHub

**Comment by damodaravadhani** [comment]
The pipeline points to 2 tests from other services saying they failed, even after I have pushed the recording for both of them.
Can someone help figure out why ? Thanks.
image

File: src/azure-cli/azure/cli/command_modules/eventhubs/commands.py [hunk 1]
```diff
@@ -177,3 +185,15 @@ def load_command_table(self, _):
...
+
+# ApplicationGroup Region
+    with self.command_group('eventhubs namespace application-group', eh_application_group_util, resource_type=ResourceType.MGMT_EVENTHUB, client_factory=application_group_mgmt_client_factory) as g:
+        g.custom_command('create', 'cli_appgroup_create')
+        g.custom_command('update', 'cli_appgroup_update')
+        g.show_command('show', 'get')
+        g.command('delete', 'delete')
+        g.command('list', 'list_by_namespace')
+
+    with self.command_group('eventhubs namespace application-group application-group-policy', eh_application_group_util, resource_type=ResourceType.MGMT_EVENTHUB, client_factory=application_group_mgmt_client_factory) as g:
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/eventhubs/commands.py:197]: I would suggest to use `eventhubs namespace application-group policy` as command group name. What do you think?

**Review by evelyn-ys — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/eventhubs/commands.py (hunk 1 above)
Comment (damodaravadhani) [reply src/azure-cli/azure/cli/command_modules/eventhubs/commands.py:197]: yes, that would be easier to use. I will make the changes and push it. Thanks !

**Review by kairu-ms — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**